test/e2e_node/device_plugin_test.go

```diff
@@ -217,7 +217,7 @@ func testDevicePlugin(f *framework.Framework, pluginSockDir string) {
 			framework.ExpectEqual(len(v1ResourcesForOurPod.Containers[0].Devices[0].DeviceIds), 1)
 		})
 
-		ginkgo.It("Keeps device plugin assignments across pod and kubelet restarts", func(ctx context.Context) {
+		ginkgo.It("Keeps device plugin assignments across pod restarts (no kubelet restart)", func(ctx context.Context) {
```

Comment:
I think it's a good idea to split the previous testcase which was testing for BOTH kubelet restart AND container (pod?) restart. We should have two `It()` blocks each corresponding to the cases, and later we should check if/how combine them.

Reply:
Sure, I can do that.